css/properties/align-items.json

```diff
@@ -8,10 +8,19 @@
             "webview_android": {
               "version_added": "4.4"
             },
-            "chrome": {
-              "version_added": "21",
-              "notes": "Older versions of the specification treat absolute positioned children as though they are a 0 by 0 flex item. Later specification versions take the children out of the flow and set their positions based on align and justify properties. Chrome implements the new behavior beginning with Chrome 52."
-            },
+            "chrome": [
+              {
+                "version_added": "52",
+                "notes": "Older versions of the specification treat absolute positioned children as though they are a 0 by 0 flex item. Later specification versions take the children out of the flow and set their positions based on align and justify properties. Chrome implements the new behavior beginning with Chrome 52."
+              },
+              {
+                "version_added": "29"
+              },
+              {
+                "prefix": "-webkit-",
+                "version_added": "21"
+              }
+            ],
             "chrome_android": {
               "version_added": null
             },
```

Comment:
`chrome_android` should be the same as `chrome`.

Reply:
Fixed